CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# Contributing
+
+1.  Fork the repository.
+
+2.  Set upstream:
+
+        $ git remote add upstream git@github.com:toknapp/arweave4s.git
+
+3.  Branch from develop:
+
+        $ git checkout develop
+        $ git checkout -b feature-x
+
+4.  Make one or more atomic commits, and ensure that each commit has a
+    descriptive commit message.
+
+5. Update existing Spec or create one.    
+
+6.  Run tests, and address any errors.
```

Comment:
The testing concept for the client is a little bit tricky. Since we need to test it on a network, we run a temp arweave network wich an modified mining speed to make integration tests possible.
You can get familiar with the step by consulting the CI config here: https://github.com/toknapp/arweave4s/blob/master/.circleci/config.yml
Feel free to describe the process here - since just run integration test would fail without running a network.

Reply:
Thank you. I was planning to explore the yml today.